examples/floating_interdomain/usecases/FloatingVl3/README.md

```diff
@@ -36,51 +36,115 @@ Make sure that you have completed steps from [interdomain](../../)
 export KUBECONFIG=$KUBECONFIG3
 ```
 
-1.2. Start **vl3 ipam** and register **vl3 network service** in the *floaing domain*.
+1.2. Start **vl3 ipam** and register **vl3 network service** in the *floating domain*.
 
 
 Note: *By default we're using ipam prefix is `169.254.0.0/16` and client prefix len is `24`. We also have two vl3 nses in this example. So we are expect to have a two vl3 addresses: `169.254.0.0` and `169.254.1.0` that should be accessible by each client.*
 
 
 ```bash
-kubectl apply -k ./cluster3
+kubectl apply -k https://github.com/networkservicemesh/deployments-k8s/examples/floating_interdomain/usecases/FloatingVl3/cluster3?ref=d8a3d2c5481d66092b877043573c268afd5c3f85
 ```
 
-1.3 Switch context to the *cluster1*.
+1.3. Get an IP address assigned to **vl3 ipam**.
+
+```bash
+vl3_ipam_ip=$(kubectl get services vl3-ipam -n nsm-system -o go-template='{{index (index (index (index .status "loadBalancer") "ingress") 0) "ip"}}')
+```
+
+1.4. Switch context to the *cluster1*.
 
 ```bash
 export KUBECONFIG=$KUBECONFIG1
 ```
 
+1.5. Prepare a patch with **vl3 ipam URL**:
+```bash
+cat > kustomization.yaml <<EOF
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+-  https://github.com/networkservicemesh/deployments-k8s/examples/floating_interdomain/usecases/FloatingVl3/cluster1?ref=d8a3d2c5481d66092b877043573c268afd5c3f85
 
-1.4. Start **vl3 nse** and client in the *cluster1*.
+patchesStrategicMerge:
+- patch-vl3-nse.yaml
+EOF
+```
 
 ```bash
-kubectl apply -k ./cluster1
+cat > patch-vl3-nse.yaml <<EOF
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nse-vl3-vpp-1
+spec:
+  containers:
+    - name: nse
+      env:
+        - name: NSM_PREFIXSERVERURL
+          value: "tcp://$vl3_ipam_ip:5006"
```

Comment:
Oops! Seems like I forgot to remove a static ip addr from the example. 

Nice catch!

I hope this would simplify the example:
```suggestion
          value: "tcp://vl3-ipam.nsm-system.cluster3:5006"
```

Reply:
Also, it would be nice if you may add `split_worlds` to the NSM_PREFIXSERVERURL env name.

Reply:
Thanks @denis-tingaikin!
Done

PR with `split_words` - https://github.com/networkservicemesh/cmd-nse-vl3-vpp/pull/107